content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -626,6 +626,12 @@ We strongly recommend using this admission controller if you intend to make use
 
 The `StorageObjectInUseProtection` plugin adds the `kubernetes.io/pvc-protection` or `kubernetes.io/pv-protection` finalizers to newly created Persistent Volume Claims (PVCs) or Persistent Volumes (PV). In case a user deletes a PVC or PV the PVC or PV is not removed until the finalizer is removed from the PVC or PV by PVC or PV Protection Controller. Refer to the [Storage Object in Use Protection](/docs/concepts/storage/persistent-volumes/#storage-object-in-use-protection) for more detailed information.
 
+### TaintNodesByCondition {#taintnodesbycondition} {{< feature-state for_k8s_version="v1.12" state="beta" >}}
+
+This admission controller automatically taints nodes based on [node conditions](/docs/concepts/architecture/nodes/#condition). The user can choose to ignore some of the node conditions by adding appropriate pod tolerations. When considering a node, scheduler looks at the node’s taints and a pod’s tolerations instead of node conditions.
```

Comment:
This all looks technically accurate. I see some style and wording nits though.

Here's some tweaked wording. @miteshskj you can take this suggestion as-is or use it to guide your own rewrite.
```suggestion
This admission controller automatically taints nodes based on [node conditions](/docs/concepts/architecture/nodes/#condition). You can tell the scheduler to ignore any of these automatic taints: add matching {{< glossary_tooltip text="tolerations" term_id="toleration" >}} to each Pod or pod template. With the TaintNodesByCondition [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) enabled (the default), the scheduler looks at the node’s taints and a pod’s tolerations instead of checking node conditions.
```

Reply:
thanks @sftim  - as requested by Huang, I have done a rewrite. Please review the latest changes.